docs/getting-started/project-setup/using-wasm-bindgen.md

```diff
@@ -1,6 +1,35 @@
 # Using wasm-bindgen
 
-{% hint style="info" %}
-`Contribute to our docs:` [Explain how to use wasm-bindgen to build an app](https://github.com/yewstack/docs/issues/34)
-{% endhint %}
+## Install
+
+```bash
+cargo install wasm-bindgen-cli
+```
+
+## Build
+
+First build the app which will generate a wasm file. Suppose you want to build the app from [build a sample app](../build-a-sample-app.md). The path of the generated file will be `target/wasm32-unknown-unknown/debug/yew-app.wasm`.
```

Comment:
We're trying to cap line lengths at no more than 100 characters – would you mind updating this PR to do that?

Reply:
Fixed. Please verify.